.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "5.0"
```

Comment:
Let's test against Node's LTS releases and the very latest non-LTS release.  LTS is currently 0.10, 0.12, 4.2; and latest release is 5.0.  I don't think we are compatible with 0.10, so we can skip that; so that leaves us 0.12, 4.2, and 5.0.